### PR TITLE
Added new fields for miam policy changes

### PR DESCRIPTION
**Pull request by khurshedk**
### Jira link (if applicable)

https://tools.hmcts.net/jira/browse/PRL-5695

### Change description ###
Miam Policy changes

### CODE REVIEW

File: docs/specs/cafcass-search-cases.json [hunk 1]
```diff
@@ -473,6 +473,188 @@
...
+          },
+          "mpuMediatorDetails": {
+            "type": "string",
+            "example": "mediator details"
+          },
+          "mpuNoDomesticAbuseEvidenceReason": {
+            "type": "string",
+            "example":"domestic abuse evidence reason"
+          },
+          "miamPolicyUpgradeTable": {
```
Comment (yogendraU) [inline docs/specs/cafcass-search-cases.json:604]: what is the relevance of this field to the api?

File: docs/specs/cafcass-search-cases.json [hunk 2]
```diff
@@ -473,6 +473,188 @@
...
+                "type": "string",
+                "example": "Yes/No"
+              },
+              "mpuChildInvolvedInMiam": {
+                "type": "string",
+                "example": "Yes/No"
+              }
+            }
+          },
+          "miamPolicyUpgradeExemptionsTable": {
```
Comment (yogendraU) [inline docs/specs/cafcass-search-cases.json:621]: mpuIsDomesticAbuseEvidenceProvided

**Review by yogendraU — CHANGES_REQUESTED** [review]

**Comment by yogendraU** [comment]
hi @khurshedk , this PR has got 6 files changed. I dont think other files apart from search cases will need a change.

File: docs/specs/cafcass-search-cases.json (hunk 1 above)
Comment (khurshedk) [reply docs/specs/cafcass-search-cases.json:604]: @yogendraU in relevance with miam table which was already there that is why kept there.

File: docs/specs/cafcass-search-cases.json (hunk 2 above)
Comment (khurshedk) [reply docs/specs/cafcass-search-cases.json:621]: @yogendraU  this is string field with Yes/No, it's also in synch with other Yes/No fields, if further clarification needed then please let me know

**Comment by khurshedk** [comment]
> hi @khurshedk , this PR has got 6 files changed. I dont think other files apart from search cases will need a change.

second commit is done in this pr by bot , which added 6 images, not sure was it as part of build.
https://github.com/hmcts/cnp-api-docs/pull/357/commits/0b86407d83655d6b1ca1d08e8e492e8fc5a8676c

File: docs/specs/cafcass-search-cases.json (hunk 1 above)
Comment (yogendraU) [reply docs/specs/cafcass-search-cases.json:604]: Please remove this field

File: docs/specs/cafcass-search-cases.json (hunk 2 above)
Comment (yogendraU) [reply docs/specs/cafcass-search-cases.json:621]: Please remove this field

**Comment by yogendraU** [comment]
Please close this PR and create a fresh one so that it wont have any other files apart from cafcass

**Comment by khurshedk** [comment]
Closing this pr as per comment

**Pull request closed without merging**